src/wp-includes/pluggable.php

```diff
@@ -2063,6 +2063,9 @@ function wp_password_change_notification( $user ) {
 		 * but check to see if it's the admin whose password we're changing, and skip this.
 		 */
 		if ( 0 !== strcasecmp( $user->user_email, get_option( 'admin_email' ) ) ) {
+
+			$switched_locale = switch_to_locale( get_admin_locale() );
```

Comment:
It would be better to leverage `switch_to_user_locale` here

Reply:
Hey, just to clarify, will this switch to the language of the user who changed their email? For example, if a user named "Bob" with the role of subscriber and language set to ES changes his password, will the to the administrator email be sent in ES, regardless of the site locale and admin locale?

Reply:
No, that wouldn‘t change.

Reply:
So this would become something like this:

```suggestion
			$admin_user = get_user_by( 'email', get_option( 'admin_email' ) );

			$switched_locale = $admin_user && switch_to_user_locale( $admin_user );
```

Reply:
is that change also necessary on line 2165?

Reply:
Yes

Reply:
With this change, when an Administrator with the locale set to DE registers a new user, and the admin_email is not associated with a registered account, it does not fall back to the default site language. Instead, it sends the email in the locale used by the Administrator who registered the new user.

Reply:
Then use switch_to_user_locale if a user exists and switch_to_locale if it doesn‘t